### ISSUE TITLE
No confirmation on contact form submissions

### ISSUE THREAD

**Issue by astrostl** [ocelotconsulting/ocelotconsulting.github.io#176]
I noticed that many of our form submissions were being received in duplicate or triplicate. Upon testing, at least on Mac/Safari, I noticed that it simply redirected to the home page after submitting with no apparent confirmation. I don't know if folks are mashing the actual submit button multiple times, or fully submitting multiple times, but I think some kind of non-redirect blocking confirmation would be useful.

**Comment by PMByrne** [comment]
Yeah. That’s what I was thinking as well. Just have a generic thank you for contacting us page that redirects after some amount of time.
